### ISSUE TITLE
Scraper "PornHub" not working anymore

### ISSUE THREAD

**Issue by PeterMueller5653** [stashapp/CommunityScrapers#1331]
Seems like pornhub has changed, this scraper is not working anymore.

**Comment by ttrv65ebyng** [comment]
Their video codes no longer start with ph sometimes, so it fails on those

**Comment by ttrv65ebyng** [comment]
I attempted by using a regex parser to change the "if ph isn't found" regex (the second one) to the following:
```
queryURLReplace:
    filename:
      - regex: (?:.*[^a-zA-Z\d])?(ph(?:[a-zA-Z\d]+)).+
        with: $1
      - regex: (?:.*[^a-zA-Z\d]?-)((?:[a-zA-Z\d]+)).+
        with: $1
```

and this seems to have worked for now for how my files are named: {TITLE_OF_VIDEO}-{ID}.{EXT}

**Comment by Maista6969** [comment]
Closing as this was fixed by #1414
